qubesadmin/tests/tools/qvm_ls.py

```diff
@@ -355,3 +363,347 @@ def test_101_list_selected(self):
             'sys-net  Running  AppVM  red    template1  sys-net\n'
             'vm1      Running  AppVM  green  template1  sys-net\n')
         self.assertAllCalled()
+
+class TC_100_Sort(qubesadmin.tests.QubesTestCase):
+    def setUp(self):
+        self.app = TestApp()
+        self.app.domains = TestVMCollection(
+            [
+                ('a', TestVM('a', label='red', maxmem='100')),
+                ('B', TestVM('B', label='green', maxmem='1000')),
+                ('c', TestVM('c', label='blue',  maxmem='300')),
+                ('dom0', TestVM('dom0', label='black', maxmem='-'))
```

Comment:
Is this maxmem value `-` legacy from your previous implementation of returning `-` if it is `0`? I still prefer integer value.

Reply:
No. It is the current existing condition. Not related to my previous (or current) implementation. This is only the unit-test BTW. I had to assure that the numeric sort works the `-`